core/src/test/scala/unit/kafka/server/ServerShutdownTest.scala

```diff
@@ -142,6 +142,52 @@ class ServerShutdownTest extends KafkaServerTestHarness {
     verifyCleanShutdownAfterFailedStartup[CancellationException]
   }
 
+  @Test
+  def testKafkaShutdownHookExecution(): Unit = {
```

Comment:
I saw the similar tests are already existed in `clients/src/test/java/org/apache/kafka/common/utils/ExitTest.java`.  Why do we need this test?

Reply:
I don't aware we already have the similar unit test for it in another place. Let me remove this one.